tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -20,64 +20,107 @@ import {backend_util, TensorInfo, util} from '@tensorflow/tfjs-core';
 import {mapActivationToShaderProgram} from './activation_util';
 import {getMainHeaderString} from './shader_preprocessor';
 import {WebGPUProgram} from './webgpu_program';
-import {computeDispatch, computeWorkGroupSizeForMatMul, tilesFitEvenlyIntoShape} from './webgpu_util';
+import {computeDispatch, computeWorkGroupSizeForMatMul} from './webgpu_util';
+
+const storeDataToSubASnippet = (transpose: boolean) => {
```

Comment:
I understand store means from register to memroy and load from memory to register. 

So how about name it loadDataToSubASnippet or readDataToSubASnippet (read is widely used)?

Also Line42，maybe loadDataFromSubASnippet=>readDataFromSubASnippet?

Reply:
Here the `store` means write, and `load` means read. If it's not clear, how about changing it to `writeDataToSubASnippet` and `readDataFromSubASnippet`? Originally, I think `storeDataToSubASnippet` and `loadDataFromSubASnippet` are the same meanings.

Reply:
Done.